### DISCUSSION TITLE
Embedding designer and dashboard into React App.

### DISCUSSION THREAD

**Discussion by JinHe-PerkinElmer** [elsa-workflows/elsa-core#4358] [Q&A]
How to integrate React App? I found a related issue: https://github.com/elsa-workflows/elsa-core/discussions/4019 and the code: https://github.com/elsa-workflows/elsa-core/tree/v3/src/designer/designer_packages/react-workspace
But the package `@elsa-workflows/elsa-workflows-designer` has been deprecated, `@elsa-workflows/elsa-workflows-designer-react-wrapper` cannot be found on the npm official website. So will this still work and is there an up-to-date way of integrating it?

**Comment by sfmskywalker** [comment]
Hello @JinHe-PerkinElmer 

Thank you for your question regarding the integration of Elsa's designer and dashboard into a React App.

To address your concerns:

- The Stencil implementation of the Elsa 3 designer will indeed be superseded by the Blazor implementation.
- This new Blazor implementation provides custom elements.
- These custom elements can be used as-is, and they can also be integrated into React applications.
- When using these custom elements in React, you have the option to wrap them inside React components for a seamless integration.
- To further assist with this, we have initiated the development of a React component library that wraps these custom elements. This will make the integration process even smoother.
- You can find more about this in the [Elsa Studio repo](https://github.com/elsa-workflows/elsa-studio).
- Additionally, here's the link to the NPM package: [Elsa Studio NPM package](https://feedz.io/org/elsa-workflows/repository/elsa-3/packages/%40elsa-workflows%2Felsa-studio-wasm-react).
- Please note that these libraries are still in development and might not be ready for production use. We are actively working on them, and any contributions or feedback from the community would be highly appreciated!

I hope this provides clarity on the matter. Let us know if you have any more questions.

**Reply by JinHe-PerkinElmer** [reply]
Thank you for your reply! I will do further investigation, we are currently preparing to integrate the V2 version, and we haven't learned about v3 in detail yet.
